index.js

```diff
@@ -19,7 +19,16 @@ import * as tarball from "./src/tarball.js";
 const kDefaultCwdOptions = { forceRootAnalysis: true, usePackageLock: true, includeDevDeps: false };
 
 export async function cwd(location = process.cwd(), options = {}, logger = new Logger()) {
-  const finalizedOptions = Object.assign({ location }, kDefaultCwdOptions, options);
+  const localRegistryURL = options.registry ? new URL(options.registry).toString() : getLocalRegistryURL();
```

Comment:
Maybe just call the variable `registry` ? (will make the code prettier i think).